staging/src/k8s.io/apiserver/pkg/admission/plugin/policy/validating/metrics/metrics.go

```diff
@@ -85,16 +89,43 @@ func newValidationAdmissionMetrics() *ValidatingAdmissionPolicyMetrics {
 	},
 		[]string{"policy", "policy_binding", "error_type", "enforcement_action"},
 	)
+	activePolicies := metrics.NewGauge(&metrics.GaugeOpts{
+		Namespace:      metricsNamespace,
+		Subsystem:      metricsSubsystem,
+		Name:           "active_policies",
+		Help:           "Validation admission policy active policies",
```

Comment:
Should this number be the same as binding num? If so, add enforcement action info inside and user could filter by enforcement action directly instead of having a separate metric?

Reply:
I believe they should be separate as there could be multiple bindings per policy, but I'm not wedded to this so can change if that's needed.